packages/hash/api/src/model/knowledge/page.model.ts

```diff
@@ -380,40 +388,19 @@ export default class extends EntityModel {
    * Get the blocks in this page.
    */
   async getBlocks(graphApi: GraphApi): Promise<BlockModel[]> {
-    const outgoingBlockDataLinks = await LinkEntityModel.getByQuery(graphApi, {
-      all: [
-        {
-          equal: [
-            { path: ["leftEntity", "uuid"] },
-            { parameter: this.getEntityUuid() },
-          ],
-        },
-        {
-          equal: [
-            { path: ["leftEntity", "ownedById"] },
-            { parameter: this.getOwnedById() },
-          ],
-        },
-        {
-          equal: [
-            { path: ["type", "versionedUri"] },
-            {
-              parameter: SYSTEM_TYPES.linkEntityType.contains.getSchema().$id,
-            },
-          ],
-        },
-        {
-          equal: [{ path: ["version"] }, { parameter: "latest" }],
-        },
-        {
-          equal: [{ path: ["archived"] }, { parameter: false }],
-        },
-      ],
+    const outgoingBlockDataLinks = await this.getOutgoingLinks(graphApi, {
+      linkEntityTypeModel: SYSTEM_TYPES.linkEntityType.contains,
     });
 
-    return outgoingBlockDataLinks.map(({ rightEntityModel }) =>
-      BlockModel.fromEntityModel(rightEntityModel),
-    );
+    return outgoingBlockDataLinks
+      .sort(
+        (a, b) =>
+          (a.getLinkMetadata().leftOrder ?? 0) -
```

Comment:
Do we want to also sort by entityId perhaps? Or version or something? It'd be nice to keep this stable until we're 100% sure there won't be overlapping orders

Reply:
Yes, that would make sense to do. Will look into that

Reply:
Done in 3ae2e562cd5c91a4de31c82cd54600e3c50442b3